packages/graphql-transformers/src/computedFieldsWhereTrasform/computedFieldsWhereTransformation.ts

```diff
@@ -1,10 +1,10 @@
-const { replace, cloneDeep, lowerFirst, isEmpty } = require('lodash');
-const { forEach, map, reduce } = require('async');
-const { getOpenCrudIntrospection, introspectionUtils } = require('@venncity/opencrud-schema-provider');
+import { replace, cloneDeep, lowerFirst, isEmpty } from 'lodash';
+import async from 'async';
```

Comment:
it's so wierd that they called their library `async` wich is a reserved keyword. I would have import it as an alias, like `asyncUtil` or something

Reply:
Agreed, changed.